packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -331,7 +331,7 @@ export interface DatabaseInstanceNewProps {
    * The number of days during which automatic DB snapshots are retained. Set
    * to zero to disable backups.
    *
-   * @default 1 day
+   * @default 1
```

Comment:
Since this is a `Duration` type then I think the @default documentation should be something like `Duration.days(1)`